GAD 8 - Binomial Heap/tests/binomilia/FastResult.java

```diff
@@ -0,0 +1,92 @@
+package tests.binomilia;
+
+import gad.binomilia.*;
+
+import java.util.*;
+
+public class FastResult implements Result {
+
+    private final List<BinomialTreeNode> currentHeap = new ArrayList<>();
+    private final List<String> logs = new ArrayList<>();
```

Comment:
It might be preferable to not store logs in graphviz format as that makes the test class itself quite hard to read. I'd suggest storing states of the heap and comparing those. If a test fails, they get turned into graphviz format on output.

Reply:
As far as I know, it is impossible to store the states of the heap from memory. If I wanted to compare it with an actual proper heap, I'd have to write the solution and compare it 1:1 with the heap from my implementation.
If the test does fail however, the student can copy paste the expected (which is displayed properly) to http://webgraphviz.com and check from there. Debugging is a part of their assignments as well, we shouldn't be spoon feeding them as well

Reply:
I understand that people should debug themselves, but having to fix every single Test because of LineEndings just seems stupid to me.
I will have a look at this, and might find a solution myself. Without that being fixed those, the tests only make somewhat sense

Reply:
Line endings have been fixed